src/ValueObject/PhpVersionFeature.php

```diff
@@ -16,6 +16,11 @@ final class PhpVersionFeature
      */
     public const ELVIS_OPERATOR = 50300;
 
+    /**
+     * @var int
+     */
+    public const DATE_TIME_INTERFACE = 50300;
```

Comment:
```suggestion
    public const DATE_TIME_INTERFACE = 50500;
```

The docs shows `>= 5.5.0`

![image](https://user-images.githubusercontent.com/924196/101262521-bd1a0b00-373f-11eb-8aa0-c117c74d0718.png)